docs/guides/reference/collection_3_summary_grid.md

```diff
@@ -11,80 +11,71 @@ Digital Earth Australia (DEA) maintains and distributes collections of
 
 * GDA94 / Australian Albers projection coordinate reference system
   (EPSG:3577),
 * Digital Earth Australia's Landsat Collection 3,
 * Digital Earth Australia's Sentinel-2 Collection,
-* With a 3-cell buffer for future growth.
+* The expanded extents of offshore islands, as detailed in the KH page: [DEA's ARD expanded processing extent](https://knowledge.dea.ga.gov.au/guides/reference/ard-expanded-processing-extent/)
 
-Shown here is an image of the new grid in EPSG:3577 projection, with tiles shown
-where we expect to have land imagery available:
+Shown below is an image of both tile grids and their extents, clipped to just the region around 
+mainland Australia so the grid sizes are visible.
 
-![](/_files/reference/collection_3_grid.jpeg)
+![](/_files/reference/collection_3_grids.png)
 
-## How can I use this grid?
+## How can I use these grids?
 
-This tiling grid scheme has been used to subdivide processing and
+These tiling grids have been used to subdivide processing and
 storage of our continental products.
-
-This tiling grid can be used as an overlay in other spatial systems to
-determine the xMyN reference of a given area of interest from within a
+These tiling grids can be used as an overlay in other spatial systems to
+determine the `xMyN` reference of a given area of interest from within a
 DEA Collection 3 summary product.
 
-Unfortunately, there is no one-to-one mapping of the new Albers
-Collection 3 grid to the old Albers Collection 2 grid.
+## Data access
 
-## Shift in origin point
+Use the links below to download the grids (as a GeoJSON), and/or view the grid extents.
 
-To accommodate an expanded area of coverage of Australia's external territories, the DEA Summary Product Grid has being shifted. The south-west origin point of the grid has been being shifted from `-5472000.0, -2688000.0` to `-6912000.0, -4416000.0` (EPSG:3577). Therefore, all tile grid references have shifted 18 tiles west and 15 tiles south. For instance, a tile reference of `x10y10` has changed to `x28y25`. For a preview, see the [provisional version of the expanded DEA Summary Product Grid](https://maps.dea.ga.gov.au/#share=s-avXJqwjUtf55qGUmweYY5KYoVnI) on DEA Maps. [Download the new grid from AWS S3](https://dea-public-data.s3.ap-southeast-2.amazonaws.com/derivative/ga_summary_grid_c3_expanded.geojson)
+[Download the Landsat grid](https://data.dea.ga.gov.au/derivative/ga_summary_grid_landsat_c3.geojson)
 
-The latest versions of all of our 'summary derivative products' have now been updated to the new origin point (but existing versions of the products were not be changed). The last product to be updated was [DEA Land Cover](/data/product/dea-land-cover-landsat/) in its version 2.0.0 release on 5 March 2025.
+[Download the Sentinel grid](https://data.dea.ga.gov.au/derivative/ga_summary_grid_sentinel_c3.geojson)
 
-## References
+[View the Landsat grid in an online map](https://maps.dea.ga.gov.au/#share=s-yUPQrYI0zfAYDldoQSqxzHjpeKx)
```

Comment:
We should add both of these grids to the DEA Maps Terria config so users can view them there too. And probably remove the link to the old non-expanded grid:

https://github.com/GeoscienceAustralia/dea-config/blob/master/prod/terria/dea-maps-v8.json#L1500-L1518